client/src/components/layout/TopBar.js

```diff
@@ -25,9 +25,10 @@ const TopBar = ({ user }) => {
     <div className="top-bar">
       <div className="top-bar-left">
         <ul className="menu">
+          <li><img src="https://i.imgur.com/ANjRDVK.png" className="logo" width="60px"/></li>
```

Comment:
no inline styling with `width`. Move that CSS property to your `logo` class in your CSS